cmake/configure/configure_2_trilinos.cmake

```diff
@@ -103,6 +103,27 @@ MACRO(FEATURE_TRILINOS_FIND_EXTERNAL var)
       SET(${var} FALSE)
     ENDIF()
 
+    #
+    # deal.II has to be configured with MPI if both Trilinos and PETSc are
+    # enabled.
+    #
+    IF( DEAL_II_WITH_TRILINOS AND DEAL_II_WITH_PETSC AND NOT DEAL_II_WITH_MPI )
```

Comment:
Would you mind removing the spaces between the argument and the opening and closing parenthesis?